GDJS/Runtime/pixi-renderers/pixi-atlas-manager.ts

```diff
@@ -0,0 +1,160 @@
+/*
+ * GDevelop JS Platform
+ * Copyright 2013-present Florian Rival (Florian.Rival@gmail.com). All rights reserved.
+ * This project is released under the MIT License.
+ */
+namespace gdjs {
+  const logger = new gdjs.Logger('Atlas Manager');
+  type AtlasManagerOnProgressCallback = (
+    loadedCount: integer,
+    totalCount: integer
+  ) => void;
+  /** The callback called when a text that was requested is loaded (or an error occurred). */
+  export type AtlasManagerRequestCallback = (
+    error: Error | null,
+    content?: pixi_spine.TextureAtlas
+  ) => void;
+
+  const atlasKinds: ReadonlyArray<string> = ['atlas'];
+  const isAtlasResource = (resource: ResourceData) =>
+    atlasKinds.includes(resource.kind);
+
+  /**
+   * AtlasManager loads atlas files with pixi loader, using the "atlas" resources
+   * registered in the game resources and process them to Pixi TextureAtlas.
+   *
+   * Contrary to audio/fonts, text files are loaded asynchronously, when requested.
+   * You should properly handle errors, and give the developer/player a way to know
+   * that loading failed.
+   */
+  export class AtlasManager {
+    private _resourcesLoader: RuntimeGameResourcesLoader;
+    private _resources: ResourceData[];
+    private _loadedAtlases: { [key: string]: pixi_spine.TextureAtlas } = {};
```

Comment:
Nitpicking: this seems a Pixi Spine specific class, so I think it makes sense to name this:
```suggestion
    private _loadedSpineAtlases: { [key: string]: pixi_spine.TextureAtlas } = {};
```